### ISSUE TITLE
How to get error line while executing ruby code in playground. 

### ISSUE THREAD

**Issue by Hareramrai** [ruby/TryRuby#80]
I was using the playground editor with below code snippet. 

````ruby
puts "welcome"
puts "Hi" + 9
puts "hello"
````

The above code produces the below error.

````
welcome
TypeError: no implicit conversion of Number into String
````

Could you help me to get the line number line while display the above error?

**Comment by hmdne** [comment]
It is possible to solve this issue, but would require porting some deal of private API from Opal

**Comment by hmdne** [comment]
Fixed by #148
